modules/google-workload-identity-federation.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-short-lived-tokens.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="google-workload-identity-federation_{context}"]
+= Google workload identity federation
```

Comment:
Should be a gerund since it's a procedure https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures